doc/user/content/sql/alter-network-policy.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "ALTER NETWORK POLICY"
+description: "`ALTER NETWORK POLICY` alters a new network policy."
+menu:
+  main:
+    parent: commands
+---
+
+{{< warn-if-unreleased-inline "v0.123.0" >}}
+{{< public-preview />}}
+
+`ALTER NETWORK POLICY` alters a new network policy.
+
+Network Policies are used to manage access to the system.
+
+## Syntax
+
+{{< diagram "alter-network-policy.svg" >}}
+{{< diagram "network-policy-rule-option.svg" >}}
+
+#### Network Policy Options 
+
+| <div style="min-width:240px">Field</div>  | Value            | Description
+|-------------------------------------------|------------------|------------------------------------------------
+| _name_                                    | `text`           | A name for the Network Policy.
+| `RULES`                                   | `text[]`         | A comma-separated list of Network Policy Rules.
+
+#### Network Policy Rule Options 
+
+| <div style="min-width:240px">Field</div>  | Value            | Description
+|-------------------------------------------|------------------|------------------------------------------------
+| _name_                                    | `text`           | A name for the Network Policy Rule.
+| `ACTION`                                  | `text`           | The action to take for this rule. `ALLOW` is the only valid option.
+| `DIRECTION`                               | `text`           | The direction of traffic the rule applies to. `INGRESS` is the only valid option.
+| `ADDRESS`                                 | `text`           | The Classless Inter-Domain Routing (CIDR) block the rule will be applied to.
+
+
+
+## Details
+Changes in network policies only take effect for new connections.
+
+An initial `default` network policy will be created. This policy allows open access to the environment and can be altered by a `superuser`.
+
+
```

Comment:
Will be nice to add an example on how that default network policy can be altered.

In the hint, when trying to drop it, you get:

```sql
│ Error: ERROR: network policy is currently in use HINT: Use ALTER SYSTEM SET 'network_policy' to change the default network policy. (SQLSTATE XX000)
```

Which is still not 100% clear how you can alter that policy.